docs/how-to/how-to-install-on-windows.md

```diff
@@ -40,9 +40,17 @@ or with whichever file location you chose to create for the config.
 
 ## OpenHIM Console
 
-### Install Nginx
+A web server will be required to host the OpenHIM Console and in this guide we will use IIS and Nginx. However any good web server will be suitable, e.g. Apache.
```

Comment:
Hmmm it might be a bit confusing with both sets of instructions now (_"do i need both iis aaand nginx?"_). Maybe we should put the nginx instructions at the bottom with note saying it's an alternative?